src/lib/builders/accordion/types.ts

```diff
@@ -1,17 +1,64 @@
+import type { Writable } from 'svelte/store';
 import type { createAccordion } from './create';
+import type { ChangeFn } from '@melt-ui/svelte/internal/helpers';
 
 type BaseAccordionProps = {
 	disabled?: boolean;
 };
 
 type SingleAccordionProps = {
-	value?: string;
+	/**
+	 * The controlled value store for the accordion. If this is not provided,
+	 * the accordion will be uncontrolled and will manage its own state. For
+	 * additional control, you can provide an `onValueChange` callback to override
+	 * the default behavior.
+	 */
+	value?: Writable<string>;
+	/**
+	 * The value of the item that should be open by default. This should only be used
+	 * when the accordion is uncontrolled.
+	 */
+	defaultValue?: string;
```

Comment:
Thoughts on making `defaultValue` an "exclusive or" with `value`? It might prevent a common footgun to enforce this exclusivity at a type-level rather than just through documentation alone.

Reply:
Yeah, I actually think this is a good idea! Don't look too far into this PR yet, just getting started :)

Reply:
Fsho—definitely not trying to step on toes! It was just top-of-mind from something else I was working on 😄